test/transform.jl

```diff
@@ -196,6 +196,34 @@ let
 end
 end
 
+@testset "correlation matrix" begin
+let
```

Comment:
The `let` block should not be needed?
```suggestion
```

Reply:
The `let` usage is consistent with the above `matrix` testset.

Reply:
Oh that's not a good example and some quite old code I guess 😄 Since you test only one specific distribution you could just remove the `for` loop as well, in addition to the `let` statement, and define `dist = ...`.

Reply:
Removed `let` and simplified the test.